External/FEXCore/Source/Interface/IR/AOTIR.cpp

```diff
@@ -137,12 +145,15 @@ namespace FEXCore::IR {
 
     std::unique_lock lk(AOTIRCacheLock);
 
-    for (auto& [String, Entry] : AOTIRCaptureCacheMap) {
+    for (auto& [StringWithPID, Entry] : AOTIRCaptureCacheMap) {
       if (!Entry.Stream) {
         continue;
       }
-
-      const auto ModSize = String.size();
+      if (!StringWithPID.ends_with(std::to_string(getpid()))) {
+        continue;
+      }
+      const auto ModSize = StringWithPID.size() - std::to_string(getpid()).size();
+      auto String = std::string(StringWithPID, 0, ModSize);
```

Comment:
Using `substr` is more explicit:
```suggestion
      auto String = StringWithPID.substr(0, ModSize);
```
This comes up a couple of times in this PR.